doc/api/fs.md

```diff
@@ -1467,6 +1467,7 @@ to be notified of filesystem changes.
 * On OS X, this uses `kqueue` for files and 'FSEvents' for directories.
 * On SunOS systems (including Solaris and SmartOS), this uses `event ports`.
 * On Windows systems, this feature depends on `ReadDirectoryChangesW`.
+* On Aix systems, this feature depends on `AHAFS`, which must be enabled.
```

Comment:
Hmm.. it would be excellent if these were links (e.g. if `kqueue' was a link to more information on `kqueue`, `event ports` was a link to more information on that, `AHAFS` was a link to more information on that, etc). That could be done in a separate PR tho.